images/ci/Dockerfile

```diff
@@ -1,12 +1,13 @@
 FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16 AS builder
 WORKDIR /go/src/github.com/openshift/cert-manager-operator
+
+# build operator
 COPY . .
 RUN make build --warn-undefined-variables
 
-FROM registry.access.redhat.com/ubi9-minimal:9.2
+FROM registry.access.redhat.com/ubi9-minimal:9.4
```

Comment:
should the base image used to be changed to rhel els?

Reply:
I checked that the EUS content image is not available on our CI registry in an attempt to not leak the private EUS content freely outside. However, I've updated midstream to use the ELS image instead of UBI and that should use the correct base image for FIPS.

Also, check-payload tool for verifying the binaries to be FIPS compliant technically still passes with UBI base as long as dynamic linking and CGO is enabled.